### PR TITLE
Adding commas in config files

### PR DESCRIPTION
**Pull request by Tanga-Jonathan**
Below is my ticket url
I have placed commas in the config files 1.5.dtd and 1.6.dtd

https://issues.openmrs.org/browse/TRUNK-6142

### CODE REVIEW

File: api/src/main/resources/org/openmrs/module/dtd/config-1.5.dtd [hunk 1]
```diff
@@ -1,110 +1,109 @@
...
-	<!ELEMENT lang (#PCDATA)>
-	<!ELEMENT file (#PCDATA)>
-
-	<!ELEMENT mappingFiles (#PCDATA)>
-	<!ELEMENT packagesWithMappedClasses (#PCDATA)>
+<!--
+Top level configuration element.
+-->
+<!ELEMENT module (
+(id),
```
Comment (ryanmccauley211) [inline api/src/main/resources/org/openmrs/module/dtd/config-1.5.dtd:6]: The indentation has been removed in this file, can you revert this please. I think the diff should only show the lines where the commas were added.

File: api/src/main/resources/org/openmrs/module/dtd/config-1.6.dtd [hunk 2]
```diff
@@ -1,118 +1,118 @@
...
-	<!ELEMENT packagesWithMappedClasses (#PCDATA)>
-	
-	<!ELEMENT conditionalResources (conditionalResource+)>
-	<!ELEMENT conditionalResource (path, (openmrsVersion | loadIfModulesPresent))>
-	<!ELEMENT path (#PCDATA)>
-	<!ELEMENT openmrsVersion (#PCDATA)>
-	<!ELEMENT loadIfModulesPresent (openmrsModule+)>
-	<!ELEMENT openmrsModule (moduleId, version)>
-	<!ELEMENT moduleId (#PCDATA)>
+Top level configuration element.
```
Comment (ryanmccauley211) [inline api/src/main/resources/org/openmrs/module/dtd/config-1.6.dtd:3]: Same with this file, please revert the removed indentation and leave only the added commas

**Review by ryanmccauley211 — CHANGES_REQUESTED** [review]
The added commas look good, it is just additional formatting that needs to be reverted please

**Comment by Tanga-Jonathan** [comment]
Okay, thank you let me put back the indentation the way it was

**Comment by Tanga-Jonathan** [comment]
I have worked on the indentation, if the changes are applicable please let
me know, and am requesting for a merge

**Review by ryanmccauley211 — APPROVED** [review]
LGTM - I also tested it locally and it fixes the new dtd tests I am trying to add

**Comment by Tanga-Jonathan** [comment]
Hello @dkayiwa am requesting that you look through, thank you

**Review by wikumChamith — CHANGES_REQUESTED** [review]
Change the topic of PR like this `TRUNK-6142 : Adding commas in config files`. Also, squash your commits into one. Can you have  look at this? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by dkayiwa** [comment]
It looks like this is already done here: https://github.com/openmrs/openmrs-core/commit/c1a84dd7bdc98ad83b7acfb23006ad3115330558

**Pull request closed without merging**